packages/react/src/contacts/Contacts.tsx

```diff
@@ -22,16 +22,10 @@ function Contacts(props: ContactsProps) {
     }, [state.auth?.currentSession?.token, state.connection.socket]);
 
     useEffect(() => {
-        if (state.userDb?.conversations) {
+        if (state.userDb?.conversations && state.userDb?.conversationsCount) {
             props.getContacts(state.connection);
         }
-    }, [state.userDb?.conversationsCount]);
-
-    useEffect(() => {
-        if (state.userDb?.conversations) {
-            props.getContacts(state.connection);
-        }
-    }, []);
+    }, [state.userDb?.conversations]);
```

Comment:
I think state.userDb?.conversationsCount has to be part of the dependency array otherwise it'll cause unusedDependecy warning